src/components/VotingResultTag/VotingResultTag.svelte

```diff
@@ -8,15 +8,25 @@
 
 	let tagColor = 'bg-purple-70 text-text-04';
 	let label = '';
+	let tagText = '';
+	let tagContainer = '';
+
+	$: if (isLarge) {
+		tagText = 'heading-compact-02';
+		tagContainer = 'px-2 py-[5px] rounded-3xl';
+	} else {
+		tagText = 'label-01';
+		tagContainer = 'px-2 py-1 rounded-3xl';
+	}
```

Comment:
We never reassign these 2 variables somewhere else so it is better to remove `let`.

```ts
$: tagText = isLarge ? 'heading-compact-02' : 'label-01';
$: tagContainer = isLarge ? 'px-2 py-[5px] rounded-3xl' : 'px-2 py-1 rounded-3xl';
```

Reply:
[d6ecc](https://github.com/wevisdemo/parliament-watch/pull/71/commits/d6ecc96e82899d51553ad9fc27788ce9818c97e5)